staticjinja/staticjinja.py

```diff
@@ -286,27 +286,27 @@ def is_static(self, filename):
         """
```

Comment:
`is_static()` above probably should be changed to `template_name` to be consistent.